library/std/src/fs.rs

```diff
@@ -283,11 +283,11 @@ pub fn read<P: AsRef<Path>>(path: P) -> io::Result<Vec<u8>> {
 ///
 /// ```no_run
 /// use std::fs;
-/// use std::net::SocketAddr;
 /// use std::error::Error;
 ///
 /// fn main() -> Result<(), Box<dyn Error>> {
-///     let foo: SocketAddr = fs::read_to_string("address.txt")?.parse()?;
+///     let address: String = fs::read_to_string("address.txt")?;
+///     println!("{}", address);
```

Comment:
Let's just call it "message" instead of "address", since the example now has nothing to do with addresses anymore.

```suggestion
///     let message: String = fs::read_to_string("message.txt")?;
///     println!("{}", message);
```

Reply:
Thank you -  good suggestion